### PR TITLE
Antispam important note to headers #740

### PR DESCRIPTION
**Pull request by J0rtIT**
Added important note on #740 

Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ In the header, look for the following headings and values.
 
 - **CAT:SPOOF** or **CAT:PHISH**: Indicates that the message appears to be spoofed, meaning that the message source cannot be validated and could be suspicious. If valid, the sender will need to make sure that they have proper SPF and DKIM configuration. Check the Authentication-Results header for more information. Although it may be difficult to get all senders to use proper email authentication methods, bypassing these checks can be extremely dangerous and is the top cause of compromises.
 
+> [!IMPORTANT]
+> A detailed list of [Anti-spam headers](../security/office-365-security/anti-spam-message-headers.md)  can be found in the previous link.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md]:
```suggestion
> For a detailed list of anti-spam headers, please see [Anti-spam message headers](../security/office-365-security/anti-spam-message-headers.md).
```

File: microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md [hunk 2]
```diff
@@ -45,6 +45,11 @@ In the header, look for the following headings and values.
...
+
+
+anti-spam-message-headers
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md]:
```suggestion
anti-spam-message-headers
```
I am not sure of the purpose of this line. Shouldn't it be deleted?

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

File: microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md (hunk 2 above)
Comment (J0rtIT) [reply microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md]: it should yeah

Comment (J0rtIT) [reply microsoft-365/compliance/prevent-email-from-being-marked-as-spam.md]: LEt me review it on code :) thank you @JohanFreelancer9

**Comment by mypil** [comment]
@chrisda - PR has been copy edited and is ready for final review. This is based on issue #740

Thank you.

cc: @kenwith

**Pull request merged**